docs/open-source/action-phrase-triggers.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 'Action Triggers'
+description: 'Activate actions with function calls or specific phrases'
+---
+## What are function calls?
+[Function calls](https://platform.openai.com/docs/guides/function-calling) allow modern Large Language Models (LLMs) like ChatGPT to perform tasks outside of text generation. 
+For example, if I create a math assistant with ChatGPT, I might make a "multiply" function that ChatGPT can call to multiply two numbers together.
+
+If you add an action to a vocode agent, it is automatically added to each ChatGPT query. If the function call is outputted by ChatGPT, the corresponding action is triggered automatically.
+
+***Note:*** *In vocode agents, ChatGPT can return a text response alongside a function call. In this scenario, the text is synthesized and played first, then the action is run.*
+
+## What are phrase triggers?
+
+Phrase triggers are text phrases that activate an action when produced by an agent. Phrase triggers are useful in production use-cases where consistency and reliability are important, since text outputs are easier to control in comparison to function calls.
+
+## Configuring action triggers
+Each vocode action config contains an `action_trigger` field  to specify how the action is triggered. The default trigger is function calling. The examples below demonstrate how to set action triggers for an `EndConversation` action config:
+
+**Function calls:**
+```python
+from vocode.streaming.action.end_conversation import EndConversationVocodeActionConfig
+from vocode.streaming.models.actions import FunctionCallActionTrigger
+
+EndConversationVocodeActionConfig(
+    type="action_end_conversation",
+    action_trigger=FunctionCallActionTrigger(
+        type="action_trigger_function_call"
+    )
+)
+```
+***Note:*** *You can also leave `action_trigger` field empty and vocode will default to function calls.*
+
+**Phrase triggers:**
```

Comment:
add a line here like: "if the agent says 'Ending conversation now', then the end conversation action will automatically be taken"